.gitlab-ci.yml

```diff
@@ -69,7 +69,7 @@ variables:
   # Integration and backend integration tests
   RUN_INTEGRATION_TESTS: "true"
   SPECIFIC_INTEGRATION_TEST: ""
-  TESTS_IN_PARALLEL: "4"
+  TESTS_IN_PARALLEL: "2"
```

Comment:
Seeing this is set for both backend, and regular integration, maybe we should split it up and have one for each, since backend is actually behaving pretty well?

Reply:
@tranchitella @kacf that was when I wanted to change to parallel factor 2, the comment from Ole makes sense, maybe we could even split it even further? (parallel factors for all test suites?)

Reply:
Yeah, it could make sense. But right now even factor 1 is not passing, so let's see if we can fix that first, and then we can see if this is still necessary.

Reply:
I created [QA-495](https://tracker.mender.io/browse/QA-495), see:
* https://github.com/mendersoftware/mender-qa/pull/640
* https://github.com/mendersoftware/integration/pull/2234
* https://github.com/mendersoftware/integration/pull/2235
* https://github.com/mendersoftware/integration/pull/2236

I plan to merge everything at once and not to make it backwards compatible.